src/os/bluestore/BitAllocator.cc

```diff
@@ -1055,12 +1055,13 @@ void BitMapAreaLeaf::init(int64_t total_blocks, int64_t area_idx,
   debug_assert(!(total_blocks % BmapEntry::size()));
 
   init_common(total_blocks, area_idx, def);
+  debug_assert(m_level == 1);
   num_child = total_blocks / pow(BitMapArea::get_span_size(), m_level);
   debug_assert(num_child);
   m_child_size_blocks = total_blocks / num_child;
 
   debug_assert(m_level == 1);
```

Comment:
@chhabaramesh Negative, I just move it ahead, as to confirm:
```
num_child = total_blocks / pow(BitMapArea::get_span_size(), m_level);
```
is identical to:

```
num_child = total_blocks /  BitMapArea::get_span_size();